src/test/ui/issue-49934.stderr

```diff
@@ -0,0 +1,24 @@
+warning: `#[derive]` on non-item statements is ignored
```

Comment:
Bikeshedding:

Better term than "non-item statement"? (That items are a subtype of statements is an idiosyncrasy of the current AST)

And:

"is ignored", "does nothing", or "is meaningless"?

Reply:
Maybe "`#[derive]` can only be applied to items"?

Reply:
That doesn't explain why it's a warning and not an error, though.